src/style.css

```diff
@@ -1 +1,77 @@
-/* add styles here */
+@font-face {
+  font-family: Arial;
```

Comment:
The font-family name 'Arial' is being redefined in the @font-face rule. Consider using a unique name for the custom font to avoid conflicts with the standard Arial font.